lib/faraday_middleware/response/parse_json.rb

```diff
@@ -7,8 +7,8 @@ class ParseJson < ResponseMiddleware
       require 'json' unless defined?(::JSON)
     end
 
-    define_parser do |body|
-      ::JSON.parse body unless body.strip.empty?
+    define_parser do |body, parser_options|
```

Comment:
since you're introducing an argument and there are other things that rely on this, i think it'd be prudent to default the block argument to an empty hash. otherwise `JSON.parse` will bomb with an error that `options is not a hash`.